sdk/nodejs/runtime/closure/v8.ts

```diff
@@ -23,14 +23,35 @@ import * as semver from "semver";
 import * as v8 from "v8";
 v8.setFlagsFromString("--allow-natives-syntax");
 
+// We depend on V8 intrinsics to inspect JavaScript functions and runtime values. These intrinsics
+// are unstable and change radically between versions. These two version checks are used by this module
+// to determine whether or not it is safe to use particular intrinsincs.
+//
+// We were broken especially badly by Node 11, which removed most of the intrinsics that we used.
+// We will need to find replacements for them.
+const isNodeV10 = semver.gte(process.version, "10.0.0");
+const isNodeV11 = semver.gte(process.version, "11.0.0");
+
+function throwUnsupportedNodeVersion(): never {
+    throw new Error(
+        `Closure serialization with Node version ${process.version} is not supported by Pulumi at this time. ` +
```

Comment:
Very minor - but suggest using "Function serialization" instead of "closure".  We try to use that in most documentation on this feature.